### PR TITLE
Travel request for Jordan Harband for TC39

### PR DESCRIPTION
**Pull request by ljharb**
This covers my flight ($2361.85) ~and my hotel (462,233¥)~ for the September TC39 meeting.

### CODE REVIEW

**Review by christian-bromann — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by joesepi — APPROVED** [review]

**Review by anonrig — APPROVED** [review]

File: programs/travel-fund/2023.md [hunk 1]
```diff
@@ -21,3 +21,4 @@
...
 | Juan       | Arboleda      | OpenJS World EU                    | Attendance + Collab Summit                                                                        | Bilbao, Spain   | September 18th - September 22nd | 2,250.00 EUR                                                         | https://github.com/openjs-foundation/community-fund/pull/33          |
 | Chengzhong | Wu            | TC39 Plenary                       | Speaking                                                                                          | Tokyo, Japan    | September 25th - September 29th | 1980 USD                                                             | https://github.com/openjs-foundation/community-fund/pull/36          |
+| Jordan     | Harband       | TC39 Plenary                       | Speaking                                                                                          | Tokyo, Japan    | September 25th - September 29th | 2,361.85 USD + 462,233¥                                              | https://github.com/openjs-foundation/community-fund/pull/39          |
```
Comment (ovflowd) [inline programs/travel-fund/2023.md:24]: @ljharb wrong PR referenced here

**Review by VoltrexKeyva — APPROVED** [review]

File: programs/travel-fund/2023.md (hunk 1 above)
Comment (ljharb) [reply programs/travel-fund/2023.md:24]: whoops, i guessed wrong

Comment (ovflowd) [reply programs/travel-fund/2023.md:24]: Hahahaha that one cracked me up.

**Comment by ljharb** [comment]
Removing draft status due to today's Standards WG approval.

**Pull request merged**